### PR TITLE
config: runtime: tests: add wifi basic test template

### PR DESCRIPTION
**Pull request by laura-nao**
Add template for WiFi basic test.

### CODE REVIEW

**Review by pawiecz — APPROVED** [review]
LGTM

(depends on using a rootfs built with changes introduced in https://github.com/kernelci/kernelci-core/commit/d46a4033097cd5a6968c5ae44015e2801c1cdf3f)

**Comment by laura-nao** [comment]
I still can't spot any wifi test in the staging instance (checked both `next` and `mainline` trees) - @pawiecz any idea on what might still be missing / wrongly configured here?

**Comment by pawiecz** [comment]
@laura-nao I verified on mainline wifi tests were run - please have a look at the Nodes I linked in https://github.com/kernelci/kernelci-pipeline/pull/655#pullrequestreview-2232608453

I'll double check `next`

File: config/runtime/tests/wifi-basic.jinja2 [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{# This test verifies WiFi basic functionality:
+ # - Checks rfkill soft block/unblock
+ # - Performs a scan for nearby networks
+ # - Checks that monitor/managed modes can be enabled
+ # This test assumes there are APs within reach.
+-#}
+- test:
+    timeout:
+      minutes: 5
```
Comment (a-wai) [inline config/runtime/tests/wifi-basic.jinja2:9]: Please allow overriding with `job_timeout` (see other test templates for reference)

**Review by a-wai — CHANGES_REQUESTED** [review]

**Comment by laura-nao** [comment]
> @laura-nao I verified on mainline that wifi tests were run - please have a look at the Nodes I linked in [kernelci/kernelci-pipeline#655 (review)](https://github.com/kernelci/kernelci-pipeline/pull/655#pullrequestreview-2232608453)
> 
> I'll double check `next`

@pawiecz I was naively checking on the Grafana dashboard, but https://github.com/kernelci/kcidb/pull/538 was not merged yet (hence no match in KCIDB). Thanks!

File: config/runtime/tests/wifi-basic.jinja2 (hunk 1 above)
Comment (laura-nao) [reply config/runtime/tests/wifi-basic.jinja2]: Right, thanks for the pointer. Done!

**Review by a-wai — APPROVED** [review]

**Pull request merged**